apps/vanilla/src/pages/1/index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BookApp</title>
+    <link rel="stylesheet" href="./css/normalize.css" />
+    <link rel="stylesheet" href="./css/variables.css" />
+    <link rel="stylesheet" href="./css/index.css" />
+    <link rel="stylesheet" href="./css/dropdown-select.css" />
+    <link rel="stylesheet" href="./css/checkbox.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet" />
+  </head>
+  <body>
+    <section class="main_section">
+      <header class="header">
+        <h1 class="header__heading">📚 Books app</h1>
+      </header>
+      <main class="main_catalog">
```

Comment:
For example in this case, the name of the wrapper section class should be `main-section`. And the name of header class should be `main-section__header`. If you want to add another block, or rename it since the name is too long, you should place your alternative name right after that wrapper block first, and then use it like the other class. In summary, you should change your class names to something like this: 
```suggestion
    <section class="main-section">
      <header class="main-section__header header">
        <h1 class="header__heading">📚 Books app</h1>
      </header>
      <main class="main_catalog">
```

Reply:
Thank you for review! Pavel said not to bother too much about BEM, because we are moving fast, and the task may take too long. But I still rewrote the styles for the header and footer a little, trying to do it according to BEM.